backend/globaleaks/utils/utility.py

```diff
@@ -82,7 +82,7 @@ def get_disk_space(path):
 
 
 def read_file(p):
-    with open(p, 'r') as f:
```

Comment:
@NCommander: please keep commits separated where possible and anticipate patches to devel when they are relevant to stabilization. :)

Reply:
The fix was not compatible with python 2.7

Proper solution compatibile with both 2.7 and 3 is to use io.open that is backported to python2.7

https://github.com/globaleaks/GlobaLeaks/commit/4e2ddee31a57ce2229c2615191a4148c84a05c2d